oecophylla/cli/launch.py

```diff
@@ -49,18 +49,54 @@ def _create_dir(_path):
         os.makedirs(_path)
 
 
+def _oeco_dir():
+    d = "%s/../../" % os.path.abspath(os.path.dirname(__file__))
+
+    if not os.path.isdir(d):
+        raise OSError('Cannot find Oecophylla install directory'
+                      '(tried %s)' % d)
+    else:
+        oeco_dir = os.path.abspath(d)
+        return oeco_dir
+
+def _setup_test():
+    d = _oeco_dir()
+
+    # set inputs
+    input_dir = os.path.join(d, 'test_data/test_reads')
+    sample_sheet = os.path.join(d,
+                        'test_data/test_config/example_sample_sheet.txt')
+    # set params
+    params = os.path.join(d,
+                          'test_data/test_config/test_params.yml')
+    envs = os.path.join(d,
+                        'test_data/test_config/test_envs.yml')
+
+    # set output
+    output_dir = os.path.join(d, 'test_out')
+
+    # prep workflow directory
+    _create_dir(output_dir)
```

Comment:
Why not directly do `os.makedirs()` since `_create_dir` is a one-line function?

Reply:
This was inherited from previous code by @mortonjt, not sure why. Presumably to allow other behavior for default directory creation?

Reply:
No problem!